evalml/tests/demo_tests/test_datasets.py

```diff
@@ -1,28 +1,47 @@
+import urllib
+
 import pandas as pd
+import pytest
 
 from evalml import demos
 
 
-def test_fraud():
-    X, y = demos.load_fraud()
+@pytest.fixture(autouse=True)
+def set_testing_headers():
+    opener = urllib.request.build_opener()
+    opener.addheaders = [("Testing", "True")]
+    urllib.request.install_opener(opener)
+
+
+@pytest.fixture(autouse=True)
+def skip_if_offline(request, set_testing_headers):
+    if request.node.get_closest_marker("skip_offline"):
```

Comment:
To make this easier to `grep`, please make the fixture name and the marker name identical